admin/autoupdater/files/usr/lib/lua/autoupdater/util.lua

```diff
@@ -1,13 +1,28 @@
-local io = io
-local math = math
 local nixio = require 'nixio'
+local fs = require 'nixio.fs'
+local util = require 'nixio.util'
 
 
-module 'autoupdater.util'
+module('autoupdater.util', package.seeall)
+
+
+-- Executes a command in the background, without parsing the command though a shell (in contrast to os.execute)
+function exec(...)
+  local pid, errno, error = nixio.fork()
+  if pid == 0 then
+    nixio.execp(...)
```

Comment:
The docs are plain wrong. The difference between exec and execp is that exec expects an absolute path, while execp respects PATH (just like the execv and execvp C functions internally used by this).